.azure_pipelines/job_templates/pytorch-template-unix.yml

```diff
@@ -33,20 +33,20 @@ jobs:
 
   # Clone PyTorch Tests repository
   - bash: |
-      B64_PAT=$(printf "%s"":$ADOTOKEN" | base64)
+      B64_PAT=$(printf "%s"":$_ADOTOKEN" | base64)
       git -c http.extraHeader="Authorization: Basic ${B64_PAT}" clone $(AZURE_DEVOPS_PYTORCH_TESTS_REPO_URL)
       cd pytorch_tests
-      git checkout mubal/exp_use_scripts
+      git checkout $(PYTORCH_TESTS_CHECKOUT_BRANCH)
     env:
-      ADOTOKEN: $(AZURE_DEVOPS_CLI_PAT)
+      _ADOTOKEN: $(AZURE_DEVOPS_CLI_PAT)
     displayName: Clone PyTorch Tests repo
 
   # Run PyTorch Unit Tests
   - bash: bash $(Build.SourcesDirectory)/pytorch_tests/scripts/linux/run.sh
     env:
-      ADOTOKEN: $(AZURE_DEVOPS_CLI_PAT)
-      AZURE_S_K: $(AZURE_STORAGE_KEY)
-      TS_CLONE_P: $(TS_CLONE_PASSWORD)
-      TS_P: $(TS_PAT)
-      TS_SM_P: $(TS_SM_PAT)
+      _ADOTOKEN: $(AZURE_DEVOPS_CLI_PAT)
+      _AZURE_S_K: $(AZURE_STORAGE_KEY)
```

Comment:
Nit: This doesn't really address the review comment. The point was to not obfuscate it as `_AZURE_S_K` if it is already referenced to the right as `AZURE_STORAGE_KEY`
So two questions: 
1. If `AZURE_STORAGE_KEY` is already available as a variable, why not use it directly? Why does it need a new name/reference?
2. If  a new name is needed, can it be `_AZURE_STORAGE_KEY: $(AZURE_STORAGE_KEY)`

Reply:
It is a secret environment variable, so it needs to be referenced directly. I've found that using a new different name such as `_AZURE_S_K` obtains the secrecret environment variable properly for use in our scripts. I'll change the name of `_AZURE_S_K` to `_AZURE_STORAGE_KEY` and make sure our env var names in our scripts are the same as the name of our env vars in our pipeline.